### ISSUE TITLE
Freeze the parameters?

### ISSUE THREAD

**Issue by tendence** [ihciah/deep-fashion-retrieval#1]
It looks like you freeze all the conv layer parameters?

**Comment by tendence** [comment]
After changing the freezing flag to true, the accuracy improves quickly.

**Comment by ihciah** [comment]
Yes, you are right.
Since the ResNet model used is pre-trained on ImageNet, the conv parameters may be general for other tasks. Training them alongside random FC layer may corrupt them. So here I first tried to train FC layer first, and set FREEZE to False and fine-tune it again.
I don't know if this method is useful or not...
